src/Modal/Modal.test.js

```diff
@@ -3,12 +3,17 @@ import { shallow, mount } from 'enzyme';
 import Modal from './Modal';
 
 describe('Modal', () => {
```

Comment:
Would it be worth to test that we have a FocusTrap element inside the Modal?

Reply:
Sure, will do.